source/19-Psalms.usfm.db

```diff
@@ -2783,7 +2783,7 @@
 \q2 pierced to the heart are the foes of the king.
 \b
 \q
-\v 6 Your throne shall endure for ever and ever
+\v 6 Your throne shall endure [us:forever|cth:for ever] and ever
```

Comment:
I think that the "for ever" here should either remain split given the fact that it is part of the phrase "for ever and ever" or that the entire "for ever and ever" be replaced with "forever".